src/app/clients/add/submitFormHelpers.ts

```diff
@@ -80,11 +75,7 @@ const deleteAdultMembers = async (
         .limit(count);
 
     if (errorExists(error, status)) {
-        // TODO VFB-22 Standardize Error Handling
-        console.error(error);
-        throw Error(
-            `Error occurred whilst updating adults in the Families table. HTTP Code: ${status}, PostgreSQL Code: ${error?.code}. `
-        );
+        throw new RequestError();
```

Comment:
Make a database error class that takes in a supabase error in the constructor so that the constructed error message can take values from that object. It's also clearer for downstream devs